fast64_internal/oot/tools/quick_import.py

```diff
@@ -0,0 +1,115 @@
+from pathlib import Path
+import os
+import re
+
+import bpy
+
+from ..f3d.properties import OOTDLImportSettings
+from ..skeleton.properties import OOTSkeletonImportSettings
+from ..animation.properties import OOTAnimImportSettingsProperty
+
+
+class QuickImportAborted(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+        self.message = message
+
+
+def quick_import_exec(context: bpy.types.Context, sym_name: str):
+    sym_name = sym_name.strip()
+    if sym_name == "":
+        raise QuickImportAborted("No symbol name given")
+    if not all(
+        (
+            "a" <= c <= "z"
+            or "A" <= c <= "Z"
+            or "0" <= c <= "9"
+            or c
+            in {
+                "_",
+            }
+        )
+        for c in sym_name
+    ):
+        raise QuickImportAborted("Symbol names only have characters a-zA-Z0-9_")
+
+    sym_def_pattern = re.compile(rf"([^\s]+)\s+{sym_name}\s*(\[[^\]]*\])?\s*=")
+
+    base_dir_p = Path(context.scene.ootDecompPath)
+    assets_objects_dir_p = base_dir_p / "assets" / "objects"
+
+    all_found_defs: dict[Path, list[tuple[str, str]]] = dict()
+
+    for dirpath, dirnames, filenames in os.walk(assets_objects_dir_p):
```

Comment:
looks like dirnames is unused, you can use ``_`` instead if you want to remove the unused thing in vscode (just a suggestion, I don't really mind if you keep the name)

Reply:
I'd rather keep it like this, these names are more or less standard usage of os.walk (copied straight from the docs) so this construct is more recognizable as is